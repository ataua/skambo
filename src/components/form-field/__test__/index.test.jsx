import React from "react";
import FormField from "../index";
import renderer from "react-test-renderer";
import { Form } from "semantic-ui-react";

import { shallow } from "enzyme";

describe("FormField Snapshot test", () => {
  it("FormField Snapshot test Render", () => {
    const tree = renderer
      .create(
        <FormField name="name" label="label" type="type" inputRef={() => {}} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

describe("Test Enzyme", () => {
  it("Rendering all arguments", () => {
    const wrapper = shallow(
      <FormField/>
    );

    expect(
      wrapper.equals(
        <Form.Field >
          <label/>
          <input/>
        </Form.Field>
      )
    ).toBeTruthy();
  });
});
