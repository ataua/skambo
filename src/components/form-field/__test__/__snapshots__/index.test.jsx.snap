// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField Snapshot test FormField Snapshot test Render 1`] = `
<div
  className="field"
>
  <label>
    <div>
      label
    </div>
    <input
      name="name"
      type="type"
    />
  </label>
</div>
`;
